geoprocessing/src/modules/planning-units-area-protected/planning-units-area-protected.worker.ts

```diff
@@ -0,0 +1,42 @@
+import { Logger, Injectable } from '@nestjs/common';
+import { Worker, QueueScheduler } from 'bullmq';
+import { join } from 'path';
+import * as config from 'config';
+
+/**
+ * @see https://docs.bullmq.io/guide/workers
+ *
+ * @debt Bullmq is expected to be supported soon in the
+ * nest.js bull wrapper. In the meanwhile we are using Bullmq
+ * in the worker
+ *
+ **/
+@Injectable()
+export class CalculatePlanningUnitAreaProtectedProcessor {
+  private readonly queueName: string = 'planning-units';
```

Comment:
Regarding both of queue names: currently in api they both refer to `planning-units-protection-level` as a name. We should use the same names, right? 
Furthermore: is this indented that we have different queues for, more or less, the same flow? (or maybe I assumed it too optimistic)

Reply:
yes i need to change that :D